### PR TITLE
set list and Add new section

### PR DESCRIPTION
**Pull request by huzaifakhan0308**
-set section for the list items
-set section for the Add new books

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -5,21 +5,99 @@
...
+  width: 100%;
+}
+
+nav h1 {
+  padding-left: 10px;
+}
+
+/* nav div {
+  padding-right: 10px;
+} */
```
Comment (Rustamxon7) [inline style.css:73]: - [ ] Please ensure that any unused commented codes are removed from the file. Thank you.

File: style.css [hunk 2]
```diff
@@ -5,21 +5,99 @@
...
+
+nav a {
```
Comment (Rustamxon7) [inline style.css:75]: [OPTIONAL] You have completed an excellent job! It is my recommendation that upon submission of a new book, the user be directed to a List page or notification detailing the recently created entry.

**Review by rustam-tolipov — CHANGES_REQUESTED** [review]
Hello @huzaifakhan0308,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
_Your project is almost complete!_
_No Linters errors_

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Check the comments under the review._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by roshan-bajgain — APPROVED** [review]
Hi @wickathou, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**